src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -65,6 +66,7 @@
 	private final BootstrapController bootstrapper;
 	private final AbstractProducer producer;
 	private RowMapBuffer rowBuffer;
+	private ConcurrentHashMap<String, RowMapBuffer> rowBufferHashMapXA = new ConcurrentHashMap();
```

Comment:
only one thread is ever mutating this, no need for `ConcurrentHashMap` -- just use a regular `HashMap`